packages/ppx/src/Property_to_runtime.re

```diff
@@ -5430,8 +5946,8 @@ let render = (~loc: Location.t, property, value, important) =>
     | Error(_) =>
       switch (render_to_expr(~loc, property, value, important)) {
       | Ok(value) => Ok(value)
-      | exception (Invalid_value(v)) =>
-        Error(`Invalid_value(value ++ ". " ++ v))
+      | Error(`Invalid_value(_))
+      | exception (Invalid_value(_)) => Error(`Invalid_value(value))
```

Comment:
In terms of behaviour, does this change anything?

Reply:
Added the Error(`Invalid_value(_)) pattern to handle the error from render_to_expr, I think we forgot to handle this when refactoring this line
https://github.com/davesnx/styled-ppx/blob/f3447e0b66fe7ec3c24216b12bb5ab8bea037f56/packages/ppx/src/Property_to_runtime.re#L5932

The exception Invalid_value pattern is redundant though, since we transformed it already on render_to_expr.

Reply:
As for the Invalid_value payload, I think I removed it at that time because it's a bit ugly lol, just pushed a commit to make it nicer.